### PR TITLE
Fix a typo (missing providing values) in a Ruby Gapic pipeline.

### PR DESCRIPTION
**Pull request by jmuk**

### CODE REVIEW

File: pipeline/tasks/gapic_tasks.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def execute(self, language, api_name, final_repo_dir):
         # Such APIs don't need packman.
         if ((language == 'ruby' or language == 'nodejs') and
                 task_utils.is_output_gcloud(final_repo_dir)):
-            return
+            return final_repo_dir
```
Comment (geigerj) [inline pipeline/tasks/gapic_tasks.py]: This logic doesn't make sense to me -- if `package_dir == final_repo_dir` always, why do we need two variables?

Comment (geigerj) [inline pipeline/tasks/gapic_tasks.py]: Ah nevermind -- I see. It is different in `protoc_tasks`

Comment (geigerj) [inline pipeline/tasks/gapic_tasks.py]: But this code can be simpler by having a single `return final_repo_dir` outside the conditional.

Comment (jmuk) [inline pipeline/tasks/gapic_tasks.py]: Done

**Comment by geigerj** [comment]
LGTM after lint errors fixed.

**Pull request merged**